File: api/src/main/java/cn/hirrao/javaee/HealthApplication.java
package cn.hirrao.javaee;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@MapperScan("cn.hirrao.javaee.mapper")
public class HealthApplication {

    public static void main(String[] args) {
        SpringApplication.run(HealthApplication.class, args);
    }

}

File: api/src/main/kotlin/cn/hirrao/javaee/HealthApplication.kt
package cn.hirrao.javaee

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
@MapperScan("cn.hirrao.javaee.mapper")
open class HealthApplication {

    fun main(args: Array<String>) {
        SpringApplication.run(HealthApplication::class.java, *args)
    }
}
